compiler/README.md

```diff
@@ -0,0 +1,53 @@
+<!-- README Document on how to build and use this Neural Network Compiler -->
+
+# Introduction
+
+This docuement discusses the ONE On-Device Neural Network compiler. The scope of this document is limited to Neural Network compiler which analyzes, quantizes and optimizes trained model. Model training, deployment & inference are NOT part of this. 
```

Comment:
Need help on if there is any specific name to the compiler?

Reply:
There is no name for our Compiler frontend.

Reply:
typo `docuement `

Reply:
`ONE On-Device Neural Network compiler` --> how about just `ONE compiler` ?

Reply:
Out compiler output is circle file, which is name for our common IR, for compiler and runtime;_onert_